auditorium/views/main.js

```diff
@@ -160,32 +175,28 @@ function view (state, emit) {
 
   var manage = !isOperator && state.model.allowsCookies
     ? html`
-      <h4>${__('Manage your data')}</h4>
-      <div class="row">
-        <div class="six columns">
-          <button class="btn u-full-width" data-role="optout" onclick="${handleOptout}">
+      <div class="card col">
+        <h4>${__('Manage your data')}</h4>
+          <a class="btn primary" data-role="optout" onclick="${handleOptout}">
             ${state.model.hasOptedOut ? __('Opt in') : __('Opt out')}
-          </button>
-        </div>
-        <div class="six columns">
-          <button class="btn u-full-width" data-role="purge" onclick="${handlePurge}">
+          </a>
+          <a class="btn primary" data-role="purge" onclick="${handlePurge}">
```

Comment:
Those need to stay buttons for accessibility reasons as they are not anchors, but buttons (i.e. you request an action to be performed, and do not follow a link).